bff-service/src/app.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import axios, { AxiosResponse, Method } from 'axios';
+import { Request } from 'express';
+
+@Injectable()
+export class AppService {
+  async proxy(serviceBaseUrl: string, req: Request): Promise<AxiosResponse> {
+    return axios({
```

Comment:
you may inject `HttpService` from `@nestjs/common` directly (built-in facade around `axios`)